voice.py

```diff
@@ -40,22 +42,19 @@ def get_type_of_audio(self):
 
 
 class SoundType(Enum):
-    TTS = 0
-    YT = 1
-    SOUND = 2
-    KIWI = 3
```

Comment:
By removing kiwi, this may break kiwi not showing up when is added to queue and reproduced on kiwi's loop. Please, keep it in mind.

Reply:
Fixed, now that type of sound is checked to determine whether to show a message or not